_shortcuts_actions/list-focus-modes.md

```diff
@@ -0,0 +1,5 @@
+---
+title: List Focus Modes
+published: true
+---
+This action returns a list of your Focus Modes, along with its icon, and the colour.
```

Comment:
Something about this reads weird to me.  The "its" feels off.  Maybe something like "… along with each Focus Modes' icon and colour"?